task/modelcar-oci-ta/0.1/modelcar-oci-ta.yaml

```diff
@@ -0,0 +1,178 @@
+      - name: BASE_IMAGE
+        value: $(params.BASE_IMAGE)
+      - name: MODEL_IMAGE
+        value: $(params.MODEL_IMAGE)
+      - name: TARGET_OCI
+        value: "/var/workdir/modelcar-oci"
+    volumeMounts:
+      - mountPath: /var/workdir
+        name: workdir
+    steps:
+      - name: use-trusted-artifact
+        image: quay.io/redhat-appstudio/build-trusted-artifacts:latest@sha256:ff35e09ff5c89e54538b50abae241a765b2b7868f05d62c4835bebf0978f3659
+        args:
+          - use
+          - $(params.SOURCE_ARTIFACT)=/var/workdir/source
+      - name: download-models
+        image: quay.io/konflux-ci/oras:latest@sha256:c68c23fe7bb1ba9fc335192761ea94fc2c9beb701f3c205890581ff62fd38d80
+        volumeMounts:
+          - mountPath: /additional-secret
+            name: additional-secret
+        script: |
+          #!/bin/bash
+          set -eu
+          set -o pipefail
+
+          oras pull "$MODEL_IMAGE" \
+            --registry-config /additional-secret/.dockerconfigjson --output /var/workdir/models
+      - name: create-modelcar-base-image
+        image: quay.io/konflux-ci/release-service-utils:2f93b7ed6a2099e7187bb110a6b95caac3b8bdbc
+        workingDir: /var/workdir
+        script: |
+          #!/bin/bash
+          set -eu
+          set -o pipefail              
+          oras copy --to-oci-layout "$BASE_IMAGE" "${TARGET_OCI}:latest"
+          chmod -R g+w "$TARGET_OCI"
+
+      - name: copy-model-files
+        image: registry.access.redhat.com/ubi9/python-311:9.5-1741020859
+        workingDir: /var/workdir
+        env:
+          - name: OLOT_VERSION
+            value: 0.1.4
+        script: |
+          #!/bin/bash
+          set -eu
+          set -o pipefail
+
+          pip install olot==${OLOT_VERSION}
+          olot -m models/README.md "$TARGET_OCI" models/*
+
+      - name: push-image
+        image: quay.io/konflux-ci/oras:latest@sha256:c68c23fe7bb1ba9fc335192761ea94fc2c9beb701f3c205890581ff62fd38d80
+        workingDir: /var/workdir
+        script: |
+          #!/bin/bash
+          set -eu
+          set -o pipefail
+
+          select-oci-auth "$IMAGE" >auth.json
+
+          echo "Pushing complete artifact manifest to ${IMAGE}"
+          oras cp --to-registry-config auth.json --from-oci-layout "${TARGET_OCI}:latest" "$IMAGE"
+
+          echo "Push complete!"
+          RESULTING_DIGEST=$(oras resolve --registry-config auth.json "${IMAGE}")
+          echo -n "$RESULTING_DIGEST" | tee "$(results.IMAGE_DIGEST.path)"
+          echo
+          echo -n "$IMAGE" | tee "$(results.IMAGE_URL.path)"
+          echo
+          echo -n "${IMAGE}@${RESULTING_DIGEST}" >"$(results.IMAGE_REF.path)"
+
+      - name: sbom-generate
+#              image: quay.io/konflux-ci/sbom-utility-scripts@sha256:1939901046f2ec0afda6d48f32dc82f991d9a4e2b4b4513635b9c79e3d4c2872
+        image: quay.io/vmugicag/build-tasks-dockerfiles@sha256:4d0d0095a04e95c5f9451d1d1ddc7908b9174ac53f9925484fb09f17e9299472
```

Comment:
This line, obviously, should not be merged. It depends on https://github.com/konflux-ci/build-tasks-dockerfiles/pull/252 being merged first before Victor can replace the image ref here.